sensors/smrtlink_sensor.py

```
@@ -0,0 +1,269 @@
+            return False
+        ccs_id = collection.get("ccsId")
+        if ccs_id and self._dataset_already_split(ccs_id):
+            return False
+        return True
+
+    # ── Trigger ────────────────────────────────────────────────
+
+    def _dispatch_trigger(self, run):
+        try:
+            collections = self._client.get(f"/smrt-link/runs/{run['uniqueId']}/collections")
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)